client/app/cloud/project/compute/tasks/onboarding/cloud-project-compute-tasks-onboarding.html

```diff
@@ -0,0 +1,22 @@
+<section class="cui-page__content">
+    <div class="text-center" data-ng-if="$ctrl.loading">
+        <oui-spinner data-size="m"></oui-spinner>
+    </div>
+    <div class="oui-slideshow oui-slideshow_onboarding oui-box" data-ng-if="!$ctrl.loading">
+        <div class="text-center my-auto">
+            <h2 class="oui-header_2" data-translate="cpc_tasks_onboarding_new"></h2>
+            <div class="oui-slideshow-panel__text">
+                <span class="oui-icon oui-icon-clock-wait" aria-hidden="true"></span>
```

Comment:
No text substitute for this icon

Reply:
It is just for cosmetic purpose

Reply:
Really ? It doesn't convey the information "please wait" ?

Reply:
I was asked to add a `metering icon`, among the ui-kit icons this one made sense given the context